test/factories/App/AppFactory.js

```diff
@@ -1,12 +1,22 @@
 const { faker, getRandomString, App } = require( '../../testHelper' );
 
-const Create = async ( { blacklists, name, admin, moderators, topUsers } ) => {
+const Create = async ( { blacklists, name, admin, moderators, topUsers, author, permlink, title } ) => {
     const data = {
         name: name || getRandomString( 10 ),
         admin: admin || faker.name.firstName().toLowerCase(),
         moderators: moderators || [],
         topUsers: topUsers || [],
-        blacklists: blacklists || { users: [], wobjects: [], posts: [], apps: [] }
+        blacklists: blacklists || { users: [], wobjects: [], posts: [], apps: [] },
+        daily_chosen_post: {
+            author: author || faker.name.firstName().toLowerCase(),
+            permlink: permlink || getRandomString(),
+            title: title || getRandomString( 20 )
+        },
+        weekly_chosen_post: {
+            author: author || faker.name.firstName().toLowerCase(),
+            permlink: permlink || getRandomString(),
+            title: title || getRandomString( 20 )
+        }
```

Comment:
edit param variables name to separate author and permlink between  daily and weekly chosen posts